spec/graphql/tracing/scout_tracing_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe GraphQL::Tracing::ScoutTracing do
+  class ScoutApm
+    module Tracer
+      def self.instrument(_, _, _)
+        "stubbed"
+      end
+    end
+  end
```

Comment:
I love the idea of stubbing the various platforms, this was how i tested them in graphql-pro, too. But is this stubbed method called anywhere? If not, can we remove it? (I guess we _need_ the modules in order to `include` them)

Reply:
Good call - I'll pull it out!